org.gimp.BaseApp.json

```diff
@@ -0,0 +1,279 @@
+{
+    "id": "org.gimp.BaseApp",
```

Comment:
I don't believe we have any setup to build two apps from one repo yet, so will have to figure out if that is what we want or just two repos.

Reply:
Actually for flathub, I could merge the BaseApp and the main manifest again if you really wanted. This was mostly to deal with nightly or other frequent builds. But flathub having only the stable release, which will be only rebuilt very rarely, it's not that much a problem if it takes a long time to rebuild anyway.

This being said, the other good thing of the BaseApp is that I share it between our nightly, dev and stable. So having some manifests based on a BaseApp while the stable one being an all-in-one will make maintainance (and my tests when I need to fix/change it) much harder.

Conclusion: I'd prefer that even on Flathub, the BaseApp to stay if possible, for maintainance sake. Yet if really that's a problem we can do without.

Reply:
Yeah, it is also a quite weird setup. I mean, i understand why you want it due to an underpowered build machine and the cache format changing over time invalidating the cache. 

However, i don't think those apply for flathuib. We have decent build machines, and the cache format changes will stop when we go 0.10 stable. I think we want the flathub json files to be as standardized as possible.